data/static.yaml

```diff
@@ -18,7 +18,7 @@ pe_status_check::checks:
   S0017: "Determines if there are any OutOfMemory errors in the Puppetdb JVM"
   S0018: "Determines if there are any OutOfMemory errors in the Orchestrator JVM"
   S0019: "Determines if there are sufficient jrubies available to serve agents"
-  S0020: "Determines"
+  S0020: "Determines if the console-service endpoint for puppet server returns true"
```

Comment:
Small thing, but the endpoint is for the Console and doesn't involve Puppet server.  So if we changed this to something like "Determines if the Console status api reports all services as running" then this should be good to go.

Reply:
Changes have been made as per Adrians recommendation